specifications/api/swagger.yaml

```diff
@@ -4768,6 +4768,18 @@ x-errExistingKeyWitnesses: &errExistingKeyWitnesses
       type: string
       enum: ['existing_key_witnesses']
 
+x-errConflictingInputResolution: &errConflictingInputResolution
+  <<: *responsesErr
+  title: conflicting_input_resolution
+  properties:
+    message:
+      type: string
+      description: |
+        Occurs when providing an input resolution which conflicts with the wallet UTxO set when balancing.
```

Comment:
I think the name `conflicting_input_resolution` is a bit confusing, because I'm not sure what a `resolution` is.

Perhaps just `conflicting_input`?

Reply:
The thing is that it's the provided (input -> output) map (essentially an UTxO set) which conflicts with the wallet UTxO set. I think `conflicting_input_resolution` is more clear than  `conflicting_input `. However, I can tweak the wording a bit...

Reply:
Can I suggest some alternatives:
- `conflicting_input`
- `conflicting_input_output_mapping`
- `conflicting_input_output_mappings`
- `conflicting_utxo`
- `conflicting_utxos`

Question: do we, on detection of an erroneous entity (of which there may be many), usually return a singular error, or a plurality?